### DISCUSSION TITLE
How to add empty sections schema to repeater?

### DISCUSSION THREAD

**Discussion by maxstevens-nl** [formkit/formkit#979] [General]
From what I could gather, in [this](https://formkit.link/c5b94892bf8c3246ef2f9b39f9f97698) example both repeaters should behave the same (show the h2 element.) However, the schema-based one doesn't do this. What am I doing wrong here?

**Comment by justin-schroeder** [comment]
Good question — you solved it the way I would have too. Unfortunately (at the moment, maybe we should change this) the `empty` section only renders if the empty slot is defined. As a workaround you can use the prefix to perform the same operation:

https://formkit.link/dcdd7a245dd8245f8a0bfdf1700aaf21

**Reply by maxstevens-nl** [reply]
Alright, that'll work for us! Thank you for your reply
